### PR TITLE
fix: Display the name and alias in Applications details page

### LINKED ISSUE #1351: Display the alias and description in the "Applications" details page
**What would you like to be added**:

In the resource table, the resource name, alias, and description are displayed, but in the resource detail interface, non-tabular display items, such as cards and pannels, do not show aliases and descriptions.

In the "Applications" details page, the alias name and description of the  services and workloads are not displayed.

**Why is this needed**:

The display of multiple cards or pannels is also a way to display the list, and the display content should be consistent with the table. 

At the same time, the alias and description make it easy for users to obtain overview information, reducing the operation of clicking on details.

If the community confirms that there is a need for improvement, I can submit a PR.

### PR DESCRIPTION
**Pull request by fukiki**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind optimization

**What this PR does / why we need it**:

In some application scenarios, users will pay attention to the alias , not only the name.

**Which issue(s) this PR fixes**:

Fixes #1351

### CODE REVIEW

File: src/pages/projects/components/Cards/Ingresses/Item.jsx [hunk 1]
```diff
@@ -42,13 +43,14 @@ const Card = ({ detail, gateway, prefix }) => {
   }
 
   const tls = detail.tls[0] || {}
+  const detailName = getDisplayName(detail) || detail.name
```
Comment (leoendless) [inline src/pages/projects/components/Cards/Ingresses/Item.jsx:46]: The fallback `|| detail.name` is unnecessary.`getDisplayName` has handled the case of no alias name.

Comment (fukiki) [reply src/pages/projects/components/Cards/Ingresses/Item.jsx]: Done.
Thanks for your review.

Comment (fukiki) [reply src/pages/projects/components/Cards/Ingresses/Item.jsx]: @leoendless Thanks for your review.

**Pull request merged**